common/djangoapps/util/enterprise_helpers.py

```diff
@@ -120,3 +124,62 @@ def insert_enterprise_pipeline_elements(pipeline):
 
     for index, element in enumerate(additional_elements):
         pipeline.insert(insert_point + index, element)
+
+
+def get_enterprise_customer_logo_url(request):
```

Comment:
To a certain extent, I understand where you're coming from. However, the current architecture means that `edx-platform` has to be aware of the internal logic of what's going on in `enterprise`. I'd still personally prefer to have as much of that logic moved to `enterprise` itself so we can have a basic stub here that calls a single `enterprise` method that we can commit to maintaining the API on.

However, I understand that merged is better than not merged, so if @mattdrayer is happy with this, I'm willing to let this one pass.